media-plugins/imlib2_loaders/imlib2_loaders-1.5.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Additional image loaders for Imlib2"
+HOMEPAGE="https://www.enlightenment.org/"
+SRC_URI="https://downloads.sourceforge.net/enlightenment/${P}.tar.gz"
+
+LICENSE="|| ( BSD GPL-2 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="eet xcf"
+
+RDEPEND="
+	>=media-libs/imlib2-${PV}
+	eet? ( dev-libs/efl[eet] )
+"
+
+src_configure() {
+	local myconf=(
+		$(use_enable eet)
+		$(use_enable xcf)
+	)
+
+	ECONF_SOURCE="${S}" \
```

Comment:
This doesn't seem to do anything, does it?

Reply:
Probably not, took it from enlightenment.eclass. Tried without and it builds and installs fine. 

I took a closer look in configure phase and this package has static configure option after all, so Ill update the ebuild accordingly and push changes. It still doesnt seem to install .pc file, even though it checks for pkgconfig, so I guess some "if ! use static-libs" is needed?
>checking for x86_64-pc-linux-gnu-pkg-config... /usr/bin/x86_64-pc-linux-gnu-pkg-config
>checking pkg-config is at least version 0.9.0... yes

Reply:
Well, unless I'm missing something this package installs plugins only, so static libraries have no value.

Reply:
Thats true. Ill ask before performing actions this time: I can safely add '--disable-static" and remove .la in every cases using
>find "${D}" -name '*.la' -delete || die

or just clean ebuild from everything related to static-libs and leave it to defaults?

Reply:
Well, I'd have to use my favorite answer: it depends™ ;-). You need to pass `--disable-static` only if the build system enables static libraries by default. If it builds only shared libs, it is unnecesary. Removing `.la` files is desired most of the time. However, some very old plugin loaders actually use them. Remove them and check if everything still works.